shell/platform/common/client_wrapper/standard_message_codec_unittests.cc

```diff
@@ -68,6 +68,18 @@ TEST(StandardMessageCodec, CanEncodeAndDecodeNull) {
   CheckEncodeDecode(EncodableValue(), bytes);
 }
 
+TEST(StandardMessageCodec, CanDecodeEmptyBytesAsNullWithoutErrorMessage) {
+  std::vector<uint8_t> bytes = {};
+  const StandardMessageCodec& codec =
+      StandardMessageCodec::GetInstance(nullptr);
+  std::stringstream stderr_buffer;
+  std::streambuf* original_stderr = std::cerr.rdbuf(stderr_buffer.rdbuf());
```

Comment:
This is a messy way to test; it's asserting that a specific logging mechanism is called rather than asserting that the read isn't happening. You should be able to instead make a trivial subclass of `StandardCodecSerializer` that asserts that `ReadValueOfType` isn't called in this scenario.

Reply:
@stuartmorgan Thank you for reviewing, I referenced other similar test codes(`common/engine_unittests.cc` and `common/rasterizer_unittests.cc`) and implemented `MockStandardCodecSerializer`.